sktime/distances/tests/_expected_results.py

```diff
@@ -13,7 +13,7 @@
     "ddtw": [0.0, 2.0884818837222006, 34.837800040564005],
     "wdtw": [12.343758137512241, 0.985380547171357, 21.265839226825413],
     "wddtw": [0.0, 1.0442409418611003, 17.418900020282003],
-    "msm": [5.0, 3.0922226886554434, None],
+    "msm": [5.0, 6.828557434224288, None],
```

Comment:
why have the expected results changed? Is it the different default c value? Can we also replace the None?